lib/fastlane/actions/backup_file.rb

```diff
@@ -0,0 +1,31 @@
+module Fastlane
+  module Actions
+    class BackupFileAction < Action
+      def self.run params
+        path = params[:path]
+        FileUtils.cp(path, "#{path}.back", {:preserve => true})
+      end
+
+      def self.description
+        'This action backup your file to "[path].back".'
+      end
+
+      def self.is_supported?(platform)
+        true
+      end
+
+      def self.author
+        "gin0606"
+      end
+
+      def self.available_options
+        [
+          FastlaneCore::ConfigItem.new(key: :path,
+                                       env_name: "",
```

Comment:
I haven't tested it yet, but I think you can just drop the `env_name` if you don't need it :+1: